<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QListWidget" name="listtodo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
    border-radius: 6px;
    border: 1px solid #bbb;
    padding: 4px;
}

QListWidget::item:hover {
    background-color: #eeeeee;
}
</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listinprogress">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>171</width>
     <height>221</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
    border-radius: 6px;
    border: 1px solid #bbb;
    padding: 4px;
}

QListWidget::item:hover {
    background-color: #eeeeee;
}
</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listdone">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>50</y>
     <width>171</width>
     <height>231</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget {
    border-radius: 6px;
    border: 1px solid #bbb;
    padding: 4px;
}

QListWidget::item:hover {
    background-color: #eeeeee;
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonaddtot">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 8px;
    padding: 6px 12px;
    border: 1px solid #ccc;
}
</string>
   </property>
   <property name="text">
    <string>todo</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnaddinprogress">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 8px;
    padding: 6px 12px;
    border: 1px solid #ccc;
}
</string>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtondone">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 8px;
    padding: 6px 12px;
    border: 1px solid #ccc;
}
</string>
   </property>
   <property name="text">
    <string>done</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSearch">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>10</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtondarkmode">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 8px;
    padding: 6px 12px;
    border: 1px solid #ccc;
}
</string>
   </property>
   <property name="text">
    <string>Darkmode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExport">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>121</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 8px;
    padding: 6px 12px;
    border: 1px solid #ccc;
}
</string>
   </property>
   <property name="text">
    <string>in txt umwandeln</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonexportpdf">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-radius: 8px;
    padding: 6px 12px;
    border: 1px solid #ccc;
}
</string>
   </property>
   <property name="text">
    <string>PDF Export</string>
   </property>
  </widget>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>401</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateEdit" name="due_date_input">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>600</y>
     <width>110</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_todo">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #ffdddd;
    color: black;
    padding: 6px;
    border-radius: 4px;
}
</string>
   </property>
   <property name="text">
    <string>To Do</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_inprogress">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #fff6cc;
    color: black;
    padding: 6px;
    border-radius: 4px;
}
</string>
   </property>
   <property name="text">
    <string>In Progress</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_done">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: #d9fdd3;
    color: black;
    padding: 6px;
    border-radius: 4px;
}
</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
